src/System.Management.Automation/engine/SpecialVariables.cs

```diff
@@ -406,5 +416,6 @@ internal enum PreferenceVariable
         Warning = 13,
         Information = 14,
         Confirm = 15,
+        PSNativeCommandUseErrorAction = 16,
```

Comment:
I think this one needs to be removed as well.

Reply:
Thanks! I added that just by following the "pattern" of pref variables but was never sure it belonged in this enum.